Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
```

Comment:
Testing reply to a reply.

---
In reply to: [58288397](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r58288397) [](ancestors = 58288397,58288395)